base.js

```diff
@@ -1,125 +1,45 @@
 module.exports = {
-  extends: 'eslint:recommended',
+  extends: 'airbnb-base',
   parser: 'babel-eslint',
   env: {
-    es6: true
+    es6: true,
   },
   parserOptions: {
-    ecmaVersion: 6
+    ecmaVersion: 6,
```

Comment:
I actually agree, but I'd rather have one well-researched and thought out source of rules rather than debate and tweak own own config